wallet/local/wallet.go

```diff
@@ -259,7 +281,7 @@ func (w *LocalWallet) Commit(transaction types.SignTxResult) (*avagoTxs.Tx, erro
 		defer cancel()
 		options := []common.Option{common.WithContext(ctx)}
 		// TODO: split error checking and recovery between issuing and waiting for status
-		issueTxErr = w.P().IssueTx(tx, options...)
+		issueTxErr = w.wallet.P().IssueTx(tx, options...)
```

Comment:
this kind of looks a bit weird, I think we should find a better naming pattern, cuz currently call wallet on wallet

Reply:
changing to primaryWallet